sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -5494,4 +5494,14 @@ ALTER TABLE users ADD COLUMN hasConsentedToGdpr boolean default false;
 ALTER TABLE users_history ADD COLUMN hasConsentedToGdpr boolean default false;
 INSERT INTO schema_version (version_number, comment) VALUES (267, 'Add gdpr popup to user');
 
+-- 2020-10-12 Add event investigation status
+ALTER TABLE events ADD COLUMN eventInvestigationStatus varchar(255) not null default 'PENDING';
```

Comment:
Also no not null constraints here; default should also be removed because it might lead to the value in the database being changed without the change date being adjusted, which could result in synchronization issues with the mobile app; we're already making sure that events are initialized as pending with the build methods